modules/sdk/node-build/esbuild/bundleJavaScriptAndCSS.mjs

```diff
@@ -0,0 +1,62 @@
+import esbuild from 'esbuild';
+import fs from 'fs/promises';
+import path from 'path';
+
+import getExactAliasPlugin from './getExactAliasPlugin.mjs';
+import getFlatName from '../util/getFlatName.mjs';
+import getEntryPoint from './getEntryPoint.mjs';
+import getExternals from './getExternals.mjs';
+import writeExportBridge from './writeExportBridge.mjs';
+import writeImportBridges from './writeImportBridges.mjs';
+
+export default async function bundleJavaScriptAndCSS(globalImports, globalSymbols, projectExports) {
+	await writeImportBridges(globalImports, globalSymbols);
+
+	await Promise.all(
```

Comment:
Compared to the file above, why `await` vs returning the promise.all

Reply:
Conceptually it's the same, however in the previous file all the function did was returning the promise and in this case it does more things. Thus if you change the order of statements or refactor this function (imagine adding more steps after the `Promise.all`) you may end up missing the `await` for the `Promise.all` (happened to me before).

TBH I think it's better to stick to using `await` even if you can return a `Promise`. It creates one promise more but I'd rather have better code readability than cutting a few nanoseconds at runtime.

Reply:
done